### PR TITLE
replacing hard coded color values with color variables

### PR DESCRIPTION
**Pull request by gvn**
There's still some hard coded values in the build, but I replaced the ones that had exact or close values in our palette.

### CODE REVIEW

File: www_src/pages/map/map.less [hunk 1]
```diff
@@ -32,20 +32,20 @@
...
-      box-shadow: 0 12px 10px -4px fade(darken(#E9EBF0, 30%), 50%);
+      box-shadow: 0 12px 10px -4px fade(darken(@lightGrey, 30%), 50%);
     }
     &.unselected {
       opacity: 0.5;
     }
     &.add {
-      background: #DCE0E6;
-      color: #C4C8CC;
+      background: darken(@lightGrey, 7%);
```
Comment (k88hudson) [inline www_src/pages/map/map.less]: This looks too dark to me now

**Comment by k88hudson** [comment]
Looks all good, except the add tiles -- those look a bit off to me

File: www_src/pages/map/map.less (hunk 1 above)
Comment (gvn) [inline www_src/pages/map/map.less]: What do you suggest?

Comment (k88hudson) [inline www_src/pages/map/map.less]: `4%` is closer to the design i think

**Comment by k88hudson** [comment]
Also, want to throw lightGrey in the background definition in main.less?

**Pull request merged**